apps/opik-backend/src/main/java/com/comet/opik/api/metrics/WorkspaceMetricResponse.java

```diff
@@ -0,0 +1,31 @@
+package com.comet.opik.api.metrics;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Builder;
+
+import java.util.List;
+import java.util.UUID;
+
+@Builder(toBuilder = true)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public record WorkspaceMetricResponse(
+        List<Result> results) {
+
+    @Builder(toBuilder = true)
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    @JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+    public record Result(UUID projectId, String name,
+            List<MetricsData> data) {
+
+        @Builder(toBuilder = true)
+        @JsonIgnoreProperties(ignoreUnknown = true)
+        @JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+        public record MetricsData(String time,
```

Comment:
I think this common from other metrics. Can we reuse the same class for all time vs value?